src/styles/main.css

```diff
@@ -0,0 +1,78 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable length-zero-no-unit */
+/* stylelint-disable rule-empty-line-before */
```

Comment:
Try to fix eslint rules and remove comments

Reply:
thank your for comment. i tried to fix this one but i downloaded extension that translates css into scss, and it is made automatically. Maybe you know how to change it ?